lang/de.json

```diff
@@ -741,6 +741,27 @@
         "ShareaholicTagSiteIdDescription": "Erstellen Sie eine neue Seite in Shareaholic und setzen Sie die Site-ID hier ein",
         "ShareaholicTagSiteIdTitle": "Shareaholic Site-ID",
         "ShowAdvancedSettings": "Erweiterte Einstellungen anzeigen",
+
+        "SiteWithoutDataReactIntro": "Das Aufzeichnen Ihrer React.js-Website oder -App ist mit Matomo einfach, selbst für Single-Page-Anwendungen.",
```

Comment:
This one might be something to be checked by a technical writer 🙈 
Reading it for me actually sounds like SPA is being devalued compared to React.js. But that is already the case in the English version, so no need to change that.

Reply:
Yeah, there were discussions whether to even have a separate React.js tab, and then PO decided to keep it and have it mostly the same as SPA/PWA with only the last step being different around the code installation.